common/djangoapps/dark_lang/middleware.py

```diff
@@ -99,6 +72,26 @@ def _fuzzy_match(self, lang_code):
                 return released_lang
         return None
 
+    def _dark_parse_accept_lang_header(self, accept):
```

Comment:
Nit: since this method doesn't actually use "self", it might be better to leave it where it originally was (outside of DarkLangMiddleware).